guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,76 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content through a local web server on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports.
+Instead, on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS web server that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optional: Set an SSL/TLS CA Credential if the web server requires it.
+* Enable the repository.
+* Synchronize the repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+
+For more information, see xref:Importing_a_Content_View_Version_{context}[].
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+
+* File repositories
+* Kickstart repositories
+* RPM repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories.
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the packages, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+* Ensure that you set the download policy to *Immediate* for all repositories within the Content View you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize products you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+.To Export a Content View Version
+* List versions of the Content View that are available for export:
++
+[subs="+quotes"]
+----
+# hammer content-view version list \
+--content-view="_My_Content_View_" \
+--organization="_My_Organization_"
+----
+
+.Export a Content View version
```

Comment:
I am unsure about this heading and the heading in line 49. Maybe use a single heading `.Procedure` instead?